### PR TITLE
Update NFU Submission Template

### PR DESCRIPTION
**Pull request by dhaselhan**
* More bug fixes!
* Flatten primary contact to keep the template simpler

### CODE REVIEW

**Review by mhuseinov — APPROVED** [review]
👍

File: services/apps/alcs/src/portal/pdf-generation/generate-submission-document.service.ts [hunk 1]
```diff
@@ -379,4 +384,12 @@ export class GenerateSubmissionDocumentService {
 
     return pdfData;
   }
+
+  private formatPid(pid?: string | null) {
+    const matches = pid?.match(/(.{3})(.{3})(.{3})/);
```
Comment (trslater) [inline services/apps/alcs/src/portal/pdf-generation/generate-submission-document.service.ts:389]: Question: I noticed at some point that there are 7-digit PID's in the database. Do we need to match those as well?

Comment (dhaselhan) [reply services/apps/alcs/src/portal/pdf-generation/generate-submission-document.service.ts:389]: There shouldn't be any, all PIDs are 9 digits. I'll take a look.

**Review by trslater — APPROVED** [review]
LGTM 👍

File: services/apps/alcs/src/portal/pdf-generation/generate-submission-document.service.ts (hunk 1 above)
Comment (dhaselhan) [reply services/apps/alcs/src/portal/pdf-generation/generate-submission-document.service.ts]: Good catch! Update the code to support any length of PIDs.

**Pull request merged**